### ISSUE TITLE
Quotes not rendering correctly

### ISSUE THREAD

**Issue by julians** [jonschlinkert/remarkable#362]
Hello,

if there’s some sort of punctuation mark after a quote, typographer won’t substitute it correctly. [Example](https://jonschlinkert.github.io/remarkable/demo/#md64=JTdCJTIyc291cmNlJTIyOiUyMkJlZ3JpZmZlJTIwJTVDJTIyQW5nc3QlNUMlMjIlMjB1bmQlMjAlNUMlMjIlQzMlODRuZ3N0ZSU1QyUyMiwlMjBkYW5uJTIwc2llaHQlMjBlcyUyMHNvJTIwYXVzOiUyMiwlMjJkZWZhdWx0cyUyMjolN0IlMjJodG1sJTIyOmZhbHNlLCUyMnhodG1sT3V0JTIyOmZhbHNlLCUyMmJyZWFrcyUyMjpmYWxzZSwlMjJsYW5nUHJlZml4JTIyOiUyMmxhbmd1YWdlLSUyMiwlMjJsaW5raWZ5JTIyOnRydWUsJTIybGlua1RhcmdldCUyMjolMjIlMjIsJTIydHlwb2dyYXBoZXIlMjI6dHJ1ZSwlMjJfaGlnaGxpZ2h0JTIyOnRydWUsJTIyX3N0cmljdCUyMjpmYWxzZSwlMjJfdmlldyUyMjolMjJodG1sJTIyJTdEJTdE):

```
Begriffe "Angst" und "Ängste", dann sieht es so aus:
->
Begriffe “Angst” und "Ängste", dann sieht es so aus:
```

Happens with commas, semicolons, fullstops, dashes etc.

**Comment by fkohrt** [comment]
Yeah, stumbled across that as well. American English has punctuation inside quotation marks, unlike British English -- or German. Hope this get's fixed!
